cmp/filter/FilterChooserModel.js

```diff
@@ -368,24 +345,16 @@ export class FilterChooserModel {
             options = [];
 
         const specs = this.fieldSpecs.filter(spec => {
-            const {filterType, displayName} = spec;
-            if (!spec.supportsOperator(operator)) return false;
-            return filterType === 'range' && this.getRegExp(queryField).test(displayName);
+            return spec.supportsOperator(operator) &&
+                spec.isRangeFilter &&
+                this.getRegExp(queryField).test(spec.displayName);
```

Comment:
Should we use `testField(spec.displayName)` here?

Reply:
done